aws_piper/aws_piper.py

```diff
@@ -1,22 +1,43 @@
 from __future__ import absolute_import
 from __future__ import print_function
+
 import json
 import os
 from datetime import datetime
 
 import yaml
 from awscli.customizations.datapipeline import translator
+from super_mario.s3_client import S3Client
 
 from .data_pipeline import DataPipelineClient
 
 
+def send_query_to_S3(cfg):
+    query_file = cfg.get('sql_query_file')
+    s3 = S3Client()
+    s3.upload_file(bucket_name=cfg.get('aws_s3_bucket'),
+                   key=cfg.get('sql_query_s3_key_dir') + query_file,
+                   path=os.path.join(os.getcwd(), query_file))
+
+
+def get_parameter_values(cfg):
+    json_values = read_json_file(cfg.get('parameter_values'))
+    json_values["values"]["mySqlActivityScriptUri"] = f"s3://{cfg.get('aws_s3_bucket')}/" \
```

Comment:
Similar comment as below. The attributes `mySqlActivityScriptUri`, `mySnsAlarmSubject`, `myScheduleStartDate` are internally defined attributes. Their names and their presence are only relevant in the context of certain pipelines. 
This is going to break, especially since you're using the square-bracket notation.

Reply:
sure, I wrote it for the specific usecase, will try to to make in aws-data-pipelines, using ".get()" or exceptions